src/generator/operations.ts

```diff
@@ -397,7 +397,8 @@ function createProtocolRequest(client: string, op: Operation, group: OperationGr
     text += '\t\treturn nil, err\n';
     text += '\t}\n';
   } else if (includeParse) {
-    text += `\tu, err := client.u.Parse(${parseVar})\n`;
+    imports.add('path');
```

Comment:
Why only for the `includeParse` case?

Reply:
The `includeParse` case is the only one where we add `client.u.Parse()`, so joining the paths will exist on all requests that want to add a path to the client url.